FirebaseAuth/Tests/Sample/ApiTests/FacebookAuthTests.m

```diff
@@ -155,8 +155,6 @@ - (NSDictionary *)createFacebookTestingAccount {
                                            error.localizedDescription);
                                  }
                                }];
-  NSString *userInfo = [[NSString alloc] initWithData:data encoding:NSUTF8StringEncoding];
-  NSLog(@"The info of created Facebook testing account is: %@", userInfo);
```

Comment:
cc @rosalyntan do you think it's worth keeping these logs?

@paulb777 maybe they're fine to keep if GHA doesn't print them, I forgot about that.

Reply:
I think it's fine to remove the log.